BattleshipLite/Program.cs

```diff
@@ -18,17 +18,17 @@ public static void Main()
             //Can only execute IsWindows() in Main
             if (OperatingSystem.IsWindows())
             {
-                SoundPlayer musicPlayer = new SoundPlayer("Background_Music.wav");
-                musicPlayer.Load();
-                musicPlayer.PlayLooping();
+                //SoundPlayer musicPlayer = new SoundPlayer("Background_Music.wav");
```

Comment:
I did comment out your sound player but only because I receive an error when trying to build the application.  It appears to be unable to find the sound file on my end.